src/qt/modaloverlay.cpp

```diff
@@ -20,19 +18,23 @@ bestHeaderDate(QDateTime()),
 layerIsVisible(false),
 userClosed(false)
 {
+
     ui->setupUi(this);
     connect(ui->closeButton, &QPushButton::clicked, this, &ModalOverlay::closeClicked);
     if (parent) {
         parent->installEventFilter(this);
         raise();
     }
-
-    blockProcessTime.clear();
+ #if defined(Q_OS_WIN)
+    sp_tray = std::shared_ptr<int> (new int(0));
+    com_r = sp_tray;
+ #endif
+   blockProcessTime.clear();
     setVisible(false);
 }
 
 ModalOverlay::~ModalOverlay()
-{
+{		
```

Comment:
you added a fair bit of trailing white space here ;-)

Reply:
this is actually failing your travis build in the lint stage – `This diff appears to have added new lines with trailing whitespace.` – https://travis-ci.org/bitcoin/bitcoin/jobs/421514716#L719

Reply:
should work if it were only unwanted thing